userspace/libscap/scap.c

```diff
@@ -355,7 +355,7 @@ scap_t* scap_open_live_int(char *error, int32_t *rc,
 					{
 						handle->syscalls_of_interest[syscall_nr] = true;
 					}
-					break;
+					// DO NOT break as some PPM_SC are used multiple times for different syscalls! (eg: PPM_SC_SETRESUID...)
```

Comment:
ouch. Still, PPM_SC_MAX isn't high enough for this to matter in practice.

Reply:
Sure. I'm just mildly concerned about the full blown quadratic nested loop here.

Reply:
Well, it isn't that high as you noticed :) moreover, it is just done at init phase therefore i guess we can turn a blind eye :D  
or we can give `libaudit` another look too! :P